### ISSUE TITLE
Check degenerate tables after sanitation update

### ISSUE THREAD

**Issue by Melkiades** [insightsengineering/tern#992]
This will highlight degenerate tables to be fixed (i.e. only `summarize_row_groups`)

**Comment by shajoezhu** [comment]
hey @Melkiades , should this be an issue in tern?

**Comment by Melkiades** [comment]
> hey @Melkiades , should this be an issue in tern?

Yes, because with the new warnings all the tables that have only summarize rows will have to be fixed
